backend/src/apiserver/client/sql.go

```diff
@@ -44,3 +45,26 @@ func CreateMySQLConfig(user, password string, mysqlServiceHost string,
 		AllowNativePasswords: true,
 	}
 }
+
+func CreatePostgreSQLConfig(user, password, postgresHost, dbName string, postgresPort uint16,
+) string {
+	var b bytes.Buffer
+	if dbName != "" {
+		fmt.Fprintf(&b, "database=%s ", dbName)
+	}
+	if user != "" {
+		fmt.Fprintf(&b, "user=%s ", user)
+	}
+	if password != "" {
+		fmt.Fprintf(&b, "password=%s ", password)
+	}
+	if postgresHost != "" {
+		fmt.Fprintf(&b, "host=%s ", postgresHost)
+	}
```

Comment:
I wonder if this is necessary, since the host address here is simply for communications inside the cluster. Could you tell me more why this is beneficial? Thank you!

Reply:
Sure thing, `hostaddr` is helpful when we are connecting to external database, for example: CloudSQL. Yes we can use `host` which is handy for `Unix-domain communication`, but `hostaddr` allows postgresql client to make `TCP/IP communication` instead.